rsp_vision/console_application/app.py

```diff
@@ -92,7 +181,9 @@ def analysis_pipeline() -> None:
     save_data(swc_blueprint_spec, folder_naming, photon_data, config)
```

Comment:
this is nice so this saves to derivatives or something according to neuroblueprint spec?